spec/preservation_ingest/update_moab_spec.rb

```diff
@@ -33,11 +32,9 @@
         allow(mock_new_version).to receive(:verify_version_storage).and_return(verification_result)
         allow(verification_result).to receive(:to_json).and_return('some_json')
         allow(verification_result).to receive(:entity).and_return(full_druid_vers)
-        allow(LyberCore::Log).to receive(:info).with('some_json')
```

Comment:
you've decided not to test for logging?  (just checking)

Reply:
Yes, unless the logging is critical for some reason, I think checking it unnecessarily bloats tests.